.github/workflows/helm-chart-ci.yaml

```
@@ -151,17 +151,19 @@ jobs:
 
       - name: Run chart-testing (install)
         run: |
+          export EXTRA_HELM_ARGS=""
           [ "$VALUES" != "default" ] && kubectl create namespace "$VALUES"
           post-install() {
             [ -x .github/tests/$VALUES/post-install.sh ] && .github/tests/$VALUES/post-install.sh
             exit $1
           }
           trap 'post-install $? $LINENO' EXIT
           [ -x .github/tests/$VALUES/pre-install.sh ] && .github/tests/$VALUES/pre-install.sh
+          [ -x .github/tests/$VALUES/settings.sh ] && source .github/tests/$VALUES/settings.sh
```

```suggestion
          echo "${EXTRA_HELM_ARGS}"
          [ -x .github/tests/$VALUES/settings.sh ] && source .github/tests/$VALUES/settings.sh
          echo "${EXTRA_HELM_ARGS}"
```

Maybe add some temporary commit with debugging lines.

You can later drop that commit with interactive rebase

```shell
git rebase -i main
```

Then in editor change `pick` to `d` for that particular commit and close the editor.